dbms/src/test/java/org/polypheny/db/misc/DataMigratorTest.java

```diff
@@ -91,39 +102,48 @@ public void partialPlacementsTest() throws SQLException {
                         + "tvarchar VARCHAR(20) NULL, "
                         + "PRIMARY KEY (tprimary) )" );
 
+                LogicalTable table = Catalog.snapshot().rel().getTable( Catalog.defaultNamespaceId, "datamigratortest" ).orElseThrow();
+
                 try {
                     statement.executeUpdate( "INSERT INTO datamigratortest VALUES (1,5,'foo')" );
 
-                    // Add data store
+                    // Add data storeId
```

Comment:
Looks like a find and replace error

Reply:
https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1520476075